### PR TITLE
Midterms interactives header switch 

### PR DESCRIPTION
**Pull request by rcrphillips**
## What does this change?
The audience team raised that we will be getting lots of non-regular traffic to our midterm results interactive next week. As such, it would be good to show the full header instead of the slim one, in order to give a better sense of the brand. For ease of development, we figured it would be best to show the full header for all interactives, rather than pinpointing the specific piece of content.

The switch is off by default but will need to be turned on before Tuesday.

## Screenshots
screen shot 2018-10-31 at 12 35 38
screen shot 2018-10-31 at 12 39 22

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -494,4 +494,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Midterms interactive header switch
+  val interactiveHeaderSwitch = Switch(
```
Comment (SiAdcock) [inline common/app/conf/switches/FeatureSwitches.scala]: Not: by convention switches have an initial capital letter.

File: common/app/model/content.scala [hunk 2]
```diff
@@ -882,7 +883,7 @@ object Interactive {
       contentType = Some(contentType),
       adUnitSuffix = section + "/" + contentType.name.toLowerCase,
       twitterPropertiesOverrides = Map( "twitter:title" -> fields.linkText ),
-      contentWithSlimHeader = true
+      contentWithSlimHeader = if(interactiveHeaderSwitch.isSwitchedOn) false else true
```
Comment (SiAdcock) [inline common/app/model/content.scala]: I think you can simplify to:

```scala
contentWithSlimHeader = !interactiveHeaderSwitch.isSwitchedOn
```

File: common/app/conf/switches/FeatureSwitches.scala [hunk 3]
```diff
@@ -494,4 +494,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Midterms interactive header switch
+  val InteractiveHeaderSwitch = Switch(
```
Comment (NataliaLKB) [inline common/app/conf/switches/FeatureSwitches.scala]: I find double negatives confusing to reason about. Could we have the safe state by `On` and then [here](https://github.com/guardian/frontend/pull/20619/files#diff-cddbf1a20ebe144ec9aa35d41a46dbb4R885) we can take off the `!` and just have `InteractiveHeaderSwitch.isSwitchedOn` instead.

**Comment by NataliaLKB** [comment]
Awesome! Nice job on this.

**Review by NataliaLKB — APPROVED** [review]
Yep that works

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**